app/models/student.rb

```diff
@@ -52,7 +52,8 @@ def senior?
 
   def number_sort
     # convert class rank to a number or it sorts as a string
-    [Student.class_ranks[class_rank], number_is_last ? 1 : 0, room_draw_number]
+    # also add leading zeros onto the room draw number so sorting as a string sorts correctly.
+    [Student.class_ranks[class_rank], number_is_last ? 1 : 0, room_draw_number.to_s.rjust(5, "0")]
```

Comment:
Please don't merge this change as part of this PR.  (Use `git cherry-pick` or `git rebase` to get the other commit on master without this).

Reply:
Is changing this still necessary since the other PR has been merged?

Reply:
Not sure.  It might be helpful to rebase off master then force push and see what changes?

Reply:
(`git pull; git rebase origin/master; git push -f`)

Reply:
I tried... no luck.  I could copy-paste the original line back in, but I think that defeats the point of having a nice clean history, right?